hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/state/migration/UniqueTokensMigratorTest.java

```diff
@@ -105,8 +105,19 @@ void givenDataSetWithVirtualMerkleData_properlyMigrated() {
     @Test
     void givenDataAlreadyMigrated_noMigration() {
```

Comment:
This test didn't check what its name implied, so I split it into two cases. First, migrate, then migrate again, check that nothing changed. Second, check that if no migration is needed, no migration actually happens.